auto_submit/lib/requests/github_webhook.dart

```diff
@@ -4,18 +4,27 @@
 
 import 'dart:async';
 import 'dart:convert';
+import 'dart:io' show Platform;
 
+import 'package:auto_submit/service/log.dart';
 import 'package:github/github.dart';
 import 'package:shelf/shelf.dart';
 
-import '../server/request_handler.dart';
+import '../service/config.dart';
+import '../service/github_service.dart';
 import '../service/log.dart';
+import '../server/request_handler.dart';
 
 /// Handler for processing GitHub webhooks.
 ///
 /// On events where an 'autosubmit' label was added to a pull request,
 /// check if the pull request is mergable and publish to pubsub.
 class GithubWebhook extends RequestHandler {
+  GithubWebhook(
+    this.config,
```

Comment:
Let's add this as a requirement for the parent class

Reply:
Gotcha Casey, makes sense to me! I will remove this to the RequestHandler class and meanwhile delete Body class.